src/commands/subcommands/projects.ts

```diff
@@ -0,0 +1,102 @@
+import { HttpStatusCode, IModify } from '@rocket.chat/apps-engine/definition/accessors';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
+import { LayoutBlock } from '@rocket.chat/ui-kit';
+
+import { TodoistApp } from '../../../TodoistApp';
+import { MiscEnum } from '../../enums/Misc';
+import {
+  getActionsBlock,
+  getButton,
+  getContextBlock,
+  getSectionBlock,
+} from '../../helpers/blockBuilder';
+import { getProjectsUrl } from '../../helpers/const';
+import { IProject } from '../../interfaces/projects';
+
+export async function projects(
```

Comment:
Hey please take a look at the suggestions I pointed out in regards to Labels, I think they are going to be usefull in here as well, since the code is really similar